### PR TITLE
refactor(ports): 💥 avoid exposing unnecessary ports and update frontend port to 3301

### PR DESCRIPTION
**Pull request by prashant-shahi**
There were users who reported running into issues because they had other applications running on the exact port(s).
If any of the ports are required by users, please feel free to expose them on your set up.

Update: Frontend port changed from `3000` to `3301`

BREAKING CHANGE:

Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

**Comment by pranay01** [comment]
this is cool 
@prashant-shahi @ankitnayan Can we also use a less common port like `3002` or `3006` for frontend instead `3000` which seems more commonly used. Also, is there any documentation/blog impact of current changes.

@ankit01-oss would need to be updated if there are

**Comment by ankitnayan** [comment]
> this is cool @prashant-shahi @ankitnayan Can we also use a less common port like `3002` or `3006` for frontend instead `3000` which seems more commonly used. Also, is there any documentation/blog impact of current changes.
> 
> @ankit01-oss would need to be updated if there are

What about `2022`?

**Comment by prashant-shahi** [comment]
@pranay01 yeah, we can do that.1

For web apps, I have seen application go with 3000, 8000, 8080, and 8888.
Since we want to avoid the collision, we can go with anything other than these.

@ankitnayan `2022` would make sense for the year, but not from the next. I think ports should be easy to remember. something like 2020, 2222, 3030, 3333, etc

**Comment by ankitnayan** [comment]
> 



> @pranay01 yeah, we can do that.1
> 
> For web apps, I have seen application go with 3000, 8000, 8080, and 8888. Since we want to avoid the collision, we can go with anything other than these.
> 
> @ankitnayan `2022` would make sense for the year, but not from the next. I think ports should be easy to remember. something like 2020, 2222, 3030, 3333, etc

Mysql - 3306, redis - 6379, rabbitmq - 5672 -> none of them easy to remember. 

@pranay01, any suggestions? Sharing a resource to check free ports https://stackoverflow.com/a/133888

**Comment by pranay01** [comment]
@prashant-shahi @ankitnayan I think you guys should take a call on this. I think the main requirement is that the ports should not conflict with other common ones. I am fine otherwise :)

**Comment by ankitnayan** [comment]
@prashant-shahi @pranay01 any preference from below list? I like `3301` and `3126`

Screenshot 2022-02-08 at 1 48 19 PM

**Comment by prashant-shahi** [comment]
from the list `3301` looks good

**Comment by ankitnayan** [comment]
Cool... Then let's keep 3301

**Comment by ankitnayan** [comment]
@prashant-shahi can you compare the file changes with [https://github.com/SigNoz/signoz/search?q=3000](https://github.com/SigNoz/signoz/search?q=3000)? I think there are still a few files that need changing

**Comment by prashant-shahi** [comment]
@ankitnayan I compared and also tested locally. It's all good.

The recent `gitpod` PR was merged to `main` instead of `develop` , causing merge conflicts. It introduced `.gitpod.yml` which has mention of port `3000` .

**Comment by pranay01** [comment]
@prashant-shahi @ankitnayan So all places in docs and blogs where port is 3000 should be updated to 3301? 
 When will this change take  effect - we have to change docs and blogs accordingly?

cc @ankit01-oss

**Comment by prashant-shahi** [comment]
@pranay01 yes, we should update in docs, recent blogs, and helm chart.

**Comment by ankitnayan** [comment]
we should try to have this in this release as part of breaking change `v0.6.0`

**Comment by pranay01** [comment]
Done

File: deploy/docker/clickhouse-setup/docker-compose.yaml [hunk 1]
```diff
@@ -69,16 +55,7 @@ services:
     volumes:
       - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
     ports:
-      - "1777:1777"   # pprof extension
-      - "8887:8888"   # Prometheus metrics exposed by the agent
-      - "14268:14268"       # Jaeger receiver
```
Comment (makeavish) [inline deploy/docker/clickhouse-setup/docker-compose.yaml:74]: @ankitnayan Have we removed support for Jaeger Receiver and all other receivers for which exposed ports are removed?

**Pull request merged**